webapp/src/lib/components/spot-component/spot-location-create-form.svelte

```diff
@@ -5,100 +5,280 @@
         Form,
         Select,
         SelectItem,
-        TextInput
+        TextInput,
+        Search,
+        Modal
     } from 'carbon-components-svelte';
+    import type { AddressResult } from '$lib/types/address/address';
+
+    const apiKey = import.meta.env.VITE_GEOCODING_API_KEY;
+
+    let searchQuery = '';
+    let debounceTimer: number | undefined;
+    let results: AddressResult[];
+    let dropdownOpen: boolean = false;
+    let warningModalOpen: boolean = false;
+    let warningText: string = '';
 
     export let isReadOnly: boolean;
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    export let handleSubmit = (_: Event) => {};
+    export let isVerified: boolean;
 
+    export let houseNumber: number;
     export let streetAddress: string;
     export let city: string;
     export let state: string;
+    export let country: string;
```

Comment:
Why do we need a country here? is country_code not enough?

Reply:
Just need it to verify the address, searching the country code wouldn't bring up the same search results.